index.js

```diff
@@ -172,15 +172,15 @@ function updateLabel(item: Element, details: Element) {
   const button = details.querySelector('[data-menu-button]')
   if (!button) return
 
-  const text = label(item) || label(item.querySelector('[data-menu-button-text]'))
-  if (text) button.textContent = text
+  const content = label(item) || label(item.querySelector('[data-menu-button-text]'))
+  if (content) button.innerHTML = content
```

Comment:
I'd prefer this attribute be renamed or we support both text and html like `data-menu-button-text` and `data-menu-button-html`.

There's a bunch of escaping security gotchas if people think the attribute is text and will be escaped by the browser, but it turns out we're using innerHTML to set the value.

I think `data-menu-button-html` is _okay_, but it annoys me a little to have html embedded in attributes.

Reply:
Okay, maybe disregard this comment. I thought the value of the `data-menu-button-text` attribute was being used, not the inner contents.

Reply:
Dang, it is lol. Okay this is confusing hehe